PurchasesCoreSwift/Purchasing/ReceiptRefreshPolicy.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ReceiptRefreshPolicy.swift
```

Comment:
I'm not sure how the header should look :/

Reply:
Totally fine with this, we should probably come up with a style guide for the header 😅

Reply:
Me either, this looks ok to me.